game.py

```diff
@@ -0,0 +1,50 @@
+""" Main module to start application"""
+from exceptions import EnemyDown
+from exceptions import GameOver
+from models import Enemy
+from models import Player
+
+
+def play():
+    """ Method to start the game"""
+    name = input("Enter your name : ")
+    global player
+    player = Player(name)
+    start = input("Enter 'start' when you are ready for battle: ")
+    if start == 'start':
+        print('Get ready')
+    else:
+        play()
+    level = 1
+    enemy = Enemy(1)
+    while True:
+        try:
+            player.attack(enemy)
+            player.defense(enemy)
+        except EnemyDown:
+            level += 1
+            player.score += 5
+            enemy = Enemy(level)
+
+
+def main():
+    """ Main method, including savegame mechanics and latest scores """
```

Comment:
You don't need this main() function.
You can totally move all of these lines to name==main block